moveit_ros/planning/trajectory_execution_manager/include/moveit/trajectory_execution_manager/trajectory_execution_manager.h

```diff
@@ -348,6 +351,7 @@ class TrajectoryExecutionManager
   double allowed_goal_duration_margin_;
   double allowed_start_tolerance_;  // joint tolerance for validate(): radians for revolute joints
   double execution_velocity_scaling_;
+  bool validate_trajectory_complete_;
```

Comment:
In #840 we discussed to have a double parameter, e.g. `allowed_goal_tolerance_`, similar to `allowed_start_tolerance_`. They should be initialized from each other if only one is specified.
However, as this parameter is dynamically reconfigurable, that's probably not feasible: they are always explicitly set by `DynamicReconfigure`.
If we keep a bool here, I suggest `wait_for_completion_`